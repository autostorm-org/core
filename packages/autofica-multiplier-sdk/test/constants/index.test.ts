import {
  WalletConnectionStates,
  SupportedNetworks,
  Contracts,
  ChainData,
} from "../../src";

describe("Constants", () => {
  test("WalletConnectionStates should match snapshots", () => {
    expect(WalletConnectionStates).toMatchSnapshot();
  });
  test("SupportedNetworks should match snapshots", () => {
    expect(SupportedNetworks).toMatchSnapshot();
  });
  test("Contracts should match snapshots", () => {
    expect(Contracts).toMatchSnapshot();
  });
  test("ChainData should match snapshots", () => {
    expect(ChainData).toMatchSnapshot();
  });
});
