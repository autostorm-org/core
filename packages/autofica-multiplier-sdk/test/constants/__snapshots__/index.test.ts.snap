// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Constants ChainData should match snapshots 1`] = `
Object {
  "56": Object {
    "name": "BSC Mainnet",
  },
  "97": Object {
    "name": "BSC Testnet",
  },
}
`;

exports[`Constants Contracts should match snapshots 1`] = `
Object {
  "56": Object {
    "BNB": "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE",
    "lendingPoolAddressProvider": "0xCc0479a98cC66E85806D0Cd7970d6e07f77Fd633",
  },
  "97": Object {
    "BNB": "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE",
    "lendingPoolAddressProvider": "0x78547CBf195Dc3D92C5847acD9E89aFB35430c0f",
  },
}
`;

exports[`Constants SupportedNetworks should match snapshots 1`] = `
Object {
  "56": "MAINNET",
  "97": "TESTNET",
  "MAINNET": 56,
  "TESTNET": 97,
}
`;

exports[`Constants WalletConnectionStates should match snapshots 1`] = `
Object {
  "0": "NOT_CONNECTED",
  "1": "METAMASK_WALLET",
  "2": "BSC_WALLET",
  "3": "NO_WALLET_DETECTED",
  "BSC_WALLET": 2,
  "METAMASK_WALLET": 1,
  "NOT_CONNECTED": 0,
  "NO_WALLET_DETECTED": 3,
}
`;
